unstructured/file_utils/filetype.py

```diff
@@ -397,6 +397,10 @@ def _detect_filetype_from_octet_stream(file: IO) -> FileType:
         elif all(f in archive_filenames for f in EXPECTED_PPTX_FILES):
             return FileType.PPTX
 
+    if LIBMAGIC_AVAILABLE:
```

Comment:
`elif`?

Reply:
It will return filetype.unk from line 407 if libmayic not available so,elif not required